src/libasr/intrinsic_func_registry_util_gen.py

```diff
@@ -1011,7 +1011,7 @@ def add_create_func_return_src(func_name):
         src += indent * 4 + "ASR::dimension_t* m_dims = nullptr;\n"
         src += indent * 4 + "size_t n_dims = ASRUtils::extract_dimensions_from_ttype(type, m_dims);\n"
         src += indent * 4 + "return_type = ASRUtils::make_Array_t_util(al, type->base.loc, "
-        src += f"{ret_type}, m_dims, n_dims, ASR::abiType::Source, false, "
+        src += f"return_type, m_dims, n_dims, ASR::abiType::Source, false, "
```

Comment:
I think we don't need this `f` now.
```suggestion
        src += "return_type, m_dims, n_dims, ASR::abiType::Source, false, "
```